test/check-application

```diff
@@ -12,7 +12,10 @@ import unittest
 TEST_DIR = os.path.dirname(__file__)
 sys.path.append(os.path.join(TEST_DIR, "common"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
 import testlib
+from testlib import skipImage
```

Comment:
Why not simply write `testlib.skipImage()`? That's already imported, and it's confusing to mix both styles.